app/assets/src/components/views/SampleViewV2/ReportTableV2.jsx

```diff
@@ -0,0 +1,404 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Table } from "~/components/visualizations/table";
```

Comment:
Nit: Can we separate these imports into 3 groups: 3rd party imports ("react", "classnames"), imports with "~" or "../..", and imports from the same directory ("./")? It's a convention we've been using in other files.